QuantConnect.TemplateBrokerage.Tests/SamcoBrokerageHistoryProviderTests.cs

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
`QuantConnect.TemplateBrokerage.Tests/` should be renamed moved all around
suggest searching `TemplateBrokerage` in the repo and removing any mention 👌